CROSSSECTOR/CPSV-AP/notes.yaml

```diff
@@ -0,0 +1,10 @@
+notesHeader:
+  These data models are mapped from [CPSV-AP v.2.2.1](https://ec.europa.eu/isa2/solutions/core-public-service-vocabulary-application-profile-cpsv-ap_en) standard.
+
```

Comment:
I've read in a previous comment that you were going to take examples from 3.0.confirm